src/java.base/share/man/java.1

```diff
@@ -3920,6 +3914,16 @@ of objects reachable from the old generation space into the young
 generation space.
 To disable GC of the young generation before each full GC, specify the
 option \f[V]-XX:-ScavengeBeforeFullGC\f[R].
+.TP
+\f[V]-Xfuture\f[R]
```

Comment:
Maybe also remove the -X help (or at least change its description) in https://github.com/openjdk/jdk/blob/c91fa278fe17ab204beef0fcef1ada6dd0bc37bb/src/java.base/share/classes/sun/launcher/resources/launcher.properties#L144

Reply:
Yes this needs to be removed from the help text.

Reply:
Thank you for spotting that Bernd. I have updated the PR to remove that text as well as updated an example program that was using `-noasyncgc`. I have searched the code for other similar occurences and haven't found anymore.